test/unit/app/femr/business/services/PatientServiceTest.java

```diff
@@ -46,7 +46,7 @@ public void setUp() throws Exception {
         mockPatientRepository = new MockPatientRepository();
         mockDataModelMapper =  new MockDataModelMapper();
         mockItemModelMapper = new MockItemModelMapper();
-        patientService = new PatientService(mockPatientRepository, mockDataModelMapper, mockItemModelMapper);
+        patientService = new PatientService(mockPatientRepository, patientEncounterRepository, mockDataModelMapper, mockItemModelMapper);
```

Comment:
This needs to be initalized before it's used, it throws an error when you run the unit tests and should be red in the IDE.

To run the tests on the CLI with SBT:  `sbt clean compile test`